### PR TITLE
neovide: 0.11.2 -> 0.12.0

### PR DESCRIPTION
**Pull request by MultisampledNight**
## Description of changes

- https://github.com/neovide/neovide/releases/tag/0.12.0

## Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [24.05 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2405.section.md) (or backporting [23.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md) and [23.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

### CODE REVIEW

File: pkgs/applications/editors/neovim/neovide/default.nix [hunk 1]
```diff
@@ -25,25 +25,25 @@
...
 
   SKIA_SOURCE_DIR =
     let
       repo = fetchFromGitHub {
         owner = "rust-skia";
         repo = "skia";
         # see rust-skia:skia-bindings/Cargo.toml#package.metadata skia
-        rev = "m113-0.61.8";
-        sha256 = "sha256-xGfkc1JLBGQW4WcblFyluZ2paEuisCVPNDU4Rfkv3BE=";
+        rev = "m113-0.67.3";
```
Comment (nekowinston) [inline pkgs/applications/editors/neovim/neovide/default.nix:45]: Did you mean [m119-0.67.3](https://github.com/rust-skia/skia/tree/m119-0.67.3)?
[m113-0.67.3 ](https://github.com/rust-skia/skia/tree/m113-0.67.3) 404's for me.

**Comment by nekowinston** [comment]
After fixing the URL for `rust-skia`, the build fails on `aarch64-darwin` for me, here are the logs:
https://gist.github.com/nekowinston/c5388ede63c5634a7d55a1e85aa7727d

File: pkgs/applications/editors/neovim/neovide/default.nix (hunk 1 above)
Comment (MultisampledNight) [reply pkgs/applications/editors/neovim/neovide/default.nix:45]: You're correct! I'll adjust it.

**Comment by MultisampledNight** [comment]
Weird! I'll try cross-compiling to aarch64-multiplatform and see if that happens too, since the log mentions about something being undefined for the arm arch.

**Comment by nekowinston** [comment]
I don't think it's aarch64, I suspect macOS specific issues like https://github.com/NixOS/nixpkgs/issues/145523

**Comment by MultisampledNight** [comment]
In this case it's complaining specifically about `SkCanvas::drawShadow` though, which was added recently and first used in this update. It didn't introduce any new deps afaik. Hmm.

**Review by ck3d — APPROVED** [review]
works fine on x86_64-linux and aarch64-linux. On aarch64-darwin the linker raises an error. But this was already the case for version 0.11.2

Thanks for updating all the dependencies.

**Comment by nekowinston** [comment]
Result of `nixpkgs-review pr 276454` run on aarch64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>neovide</li>
  </ul>
</details>

**Comment by MultisampledNight** [comment]
@ck3d 

> works fine on x86_64-linux and aarch64-linux. On aarch64-darwin the linker raises an error. But this was already the case for version 0.11.2

What was the error on 0.11.2, by chance? Would be interesting for reference

**Comment by nekowinston** [comment]
v0.11.2 build logs on aarch64-darwin: https://gist.github.com/nekowinston/14333f4e8157cab2422df2b0c5685b50

**Comment by nekowinston** [comment]
> works fine on x86_64-linux and aarch64-linux. On aarch64-darwin the linker raises an error. But this was already the case for version 0.11.2

Shouldn't this package get marked as broken if this is an expected build failure? @ck3d

**Comment by ck3d** [comment]
Yes, please remove Darwin from supported systems.

**Comment by MultisampledNight** [comment]
Unhappy about doing so, but since it's difficult to debug something one doesn't have access to I guess it's necessary. Did that.

**Review by ck3d — APPROVED** [review]

**Comment by n8henrie** [comment]
https://github.com/NixOS/nixpkgs/pull/279185

**Pull request merged**